lib.php

```diff
@@ -81,3 +84,34 @@ function tool_mergeusers_build_quiz_options() {
 
     return $result;
 }
+
+/**
+ * Profile callback to add merging data to a users profile.
+ *
+ * @param tree $tree Tree object
+ * @param stdClass $user user object
+ * @param bool $iscurrentuser
+ * @param null|stdClass $course Course object
+ */
+function tool_mergeusers_myprofile_navigation($tree, $user, $iscurrentuser, $course) {
+    if (has_capability('tool/mergeusers:mergeusers', context_system::instance())) {
```

Comment:
I would define another different capacity.

It is a good practice in general to have a capacity for viewing things and a different capacity for making this "things".

For instance, in our institution, there are support staff that cannot merge users, but would be good for them to have access to this information. 

Capacity name could be something like `tool/mergeusers:viewlog`.

Reply:
Agreed, this is a good idea - have added